app/src/interfaces/list/list.vue

```diff
@@ -168,9 +168,10 @@ const defaults = computed(() => {
 
 const fieldsWithNames = computed(() =>
 	props.fields?.map((field) => {
+		const name = field.name ?? field.field!;
 		return {
 			...field,
-			name: formatTitle(field.name ?? field.field!),
+			name: name.startsWith('$t:') ? name : formatTitle(name),
```

Comment:
Should we translate the name prior to `formatTitle()`?

Reply:
I don't think so. The `formatTitle()` call is intended for field names only because there you cannot use spaces and so on. On the other hand, translations should already be formatted correctly.
We could however limit the `startsWith('$t:')` check to `field.name` only...

For the context:
<img width="500" alt="Screenshot 2023-05-25 at 09 15 30" src="https://github.com/directus/directus/assets/5363448/4e7b3944-1c7c-4894-ac5d-c1ed3906f677">